job-roles/lead-engineer.md

```diff
@@ -1,57 +1,51 @@
-
-### Customer/User engagement
-This is about expectations on engaging with users/customer support.
-- Make sure required trainings happened with support team
-- Make sure dev/E2E teams have enough time in their tasks by empowering support team in solving user/customer issues
-- Avoid any escalations from customers with timely and apt response
-- Review RCAs and improve its format to convert them to product backlog
-- Being active and leading few interested feature areas in other related opensource projects
-
-### Social media engagement
-This is about building self technical image on social media which in-turn increases that of company.
-- Completing surveys
-- Strong advocacy of component you worked in other mediums like K8s channels, or other opersource channels
-- Lead the discussion on social media like reddit, stackoverflow, twitter etc
-- Inform world about our persona with clear examples
-- Seminars/proposals on regular basis on various platform about company's interested personas, product
-- Tweets about self-learnings, company, product
+# Lead Engineer / Team Lead
+
+{{#include intro.md}}
+
+## In the nutshell
+
+Lead Engineer aka Team Lead is a core leadership role in the MayaData. You are responsible for a particular team of about 8 people, working on delivering component of the MayaData portfolio.
+
+Team Lead is still considered a technical position. You are expected to be coding and contributing at more then 50% of your time. We don't believe in dedicated people managers on the team level as we are a strong engineering organization.
+
+## Responsibilities
+### Team Leadership
+- Make sure, that people in the team are feeling comfortable and secure. They understand overall MayaData vision and strategy. They understand **why** they are working on particular features.
+- Understand the gaps by involving with multiple teams in company.
+- Suggest improvements and changes to the overall organization of the Engineering and your team.
+- Identify metrics for various components like code quality, team productivity, motivations, culture, sessions feedback etc.
+- Define goals/[OKRs](/process/okr.md) with and for your team.
+- Help you team to learn and grow, identifying external/internal training requirements.
+- Exhibit and inspire team to follow process/culture.
+- Track our marketing and sales activities. Encourage participation and participate at conferences, write blog articles, tweet etc.
+
+### Project Management
+- Keeping an eye on your team's JIRA and keeping it healthy.
+- Project level tracking for execution completion within time.
+- Lead/partcipate in internal project review meeting.
+- Helping in risk mitigation for project execution.
+- Plan team size in advance.
+- Project level check for required E2E coverage.
+- Make sure that we follow team's definition of done.
+
+### Product Owner
+- Convert Stories to backlog Epics and Tasks. Participate at grooming sessions with Product Management.
```

Comment:
Should we call out here if this includes the ownership of meeting release milestones?

Reply:
OK, let me propose something here.

Reply:
cool!